plugin/mouse/mouse.py

```diff
@@ -52,6 +52,12 @@
     default=0,
     desc="Pop noise clicks left mouse button. 0 = off, 1 = on with eyetracker but not with zoom mouse mode, 2 = on but not with zoom mouse mode",
 )
+setting_mouse_enable_pop_wake = mod.setting(
+    "mouse_enable_pop_wake",
+    type=int,
```

Comment:
```suggestion
    type=bool,
```

Made it a suggestion for easier application :)